atlp-rwanda/Team-furebo-E-commerce-fn
# Ft-chat-functionality-for-users-185172193

What does this PR do?
Add chat functionality for users

Description of Task to be completed?

This pull request implements the chat functionality for users, allowing them to interact with each other on the platform. By enabling chat features, users can ask questions, share recommendations, etc.

How should this be manually tested?

1. Ensure the user is online and logged into the platform.
2. Click on the chat icon and verify that all past chats are displayed.
3. Post a public chat message

What are the relevant pivotal tracker/Trello stories?
#185172193
